src/app/ApplicationTemplate.Shared.Views/Behaviors/MenuMarginBehavior.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Microsoft.UI.Xaml;
+
+namespace ApplicationTemplate.Views.Behaviors
+{
+	/// <summary>
+	/// This behaviors adjusts the margin of the control to set the height of the menu
+	/// when IsMarginEnabled == true. This way the page content is not hidden by the menu.
+	/// </summary>
+	public class MenuMarginBehavior
+	{
+		/// <summary>
```

Comment:
I need to either prove that tabbar is always 62 or make sure we get the actual height of the tabbar